test/e2e/revision_test.go

```diff
@@ -59,6 +59,9 @@ func TestRevision(t *testing.T) {
 	nonexistRevision := "hello-nonexist"
 	test.RevisionMultipleDelete(r, existRevision1, existRevision2, nonexistRevision)
 
+	t.Log("delete all revision from hello service and return no error")
+	test.RevisionDeleteWithAllOption(r, "hello")
+
```

Comment:
At this point, following the order of commands in this test, the service will have only one revision. You can update the service yet again to increase the count to 2 and then try deleting them all.